vendor/github.com/grantseltzer/golumns/columns.go

```diff
@@ -40,6 +40,9 @@ func fewestColumns(lengths []int) int {
 
 func columnLength(lengths []int) int {
 	sort.Ints(lengths)
+	if len(lengths) == 0 {
+		return 4
```

Comment:
Just re-read your comment, that's a good idea, I'll include it in the next update to golumns, working on another patch to it right now anyway.